spec/features/questions/new_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+require 'pry'
+RSpec.describe "display all questions" do
```

Comment:
This description doesn't match what the test does.

Reply:
changed to "display question form"